src/tcgwars/logic/impl/gen3/LegendMaker.groovy

```diff
@@ -1066,7 +1066,7 @@ public enum LegendMaker implements LogicCardInfo {
           text "As long as Spinda is your Active Pokémon, whenever your opponent's Basic Pokémon tries to attack, your opponent flips a coin. If tails, that attack does nothing. You can't use more than 1 Pattern Distraction Poké-Body each turn."
           def endTurn = false
           delayedA {
-            before CHECK_ATTACK_REQUIREMENTS, {
+            before PROCESS_ATTACK_EFFECTS, {
```

Comment:
as far as I know PROCESS_ATTACK_EFFECTS  step shouldn't be prevented, does it work properly? 

maybe this is better

```          
delayedA (priority: BEFORE_LAST) {
              before CHECK_ATTACK_REQUIREMENTS, {
```

Reply:
Fixed below, thanks for the suggestion 👍 (Also helped solve a problem with Cursed Shovel and Spell Tag, though that needs more work in general (alongside Max Phantom-like attacks)

Reply:
@starg09 Good one, I think it'd makes sense to change priority of energy cost check inside `Attack` to `AFTER_FIRST`  later on.